packages/core/src/index.ts

```diff
@@ -48,7 +48,6 @@ export * as IO from "./IO/index.js"
 export * as Id from "./Id/index.js"
 export * as Identity from "./Identity/index.js"
 export * as IndexedT from "./IndexedT/index.js"
-export * as InvariantT from "./InvariantT/index.js"
```

Comment:
@mikearnaldi Removed InvariantT, unless I'm mistaken this isn't needed with the new system